CHANGES.rst

```diff
@@ -14,6 +14,13 @@ documentation
 
 - Update tutorial usage of ``jump.threshold`` to ``jump.rejection_threshold``. [#7572]
 
+
+extract_1d:
```

Comment:
This entry is not formatted correctly. Get rid of the colon at the end of the step name and indent the following text lines, preceded by a dash "-"  (as with all other log entries)